src/tests/functional/plugin/shared/src/behavior/plugin/life_time.cpp

```diff
@@ -38,9 +38,6 @@ namespace BehaviorTestsDefinitions {
         ::testing::GTEST_FLAG(death_test_style) = deathTestStyle;
```

Comment:
Shouldn't we get rid of any deathTestStyle variables/related code if we are going not to use GTest death test functionality?